ResearchKit/ActiveTasks/ORKAudioContentView.m

```diff
@@ -356,7 +362,7 @@ - (void)updateGraphSamples {
 
 - (void)updateAlertLabelHidden {
     NSNumber *sample = [_samples lastObject];
-    BOOL hide = _finished || !([sample doubleValue] > _alertThreshold);
+    BOOL hide = _finished || !_failed || (sample && !([sample doubleValue] > _alertThreshold));
```

Comment:
This condition is not right, and also hard to parse. I'd prefer:

BOOL show = (! _finished && ([sample doubleValue] > _alertThreshold)) || _failed;
_alertLabel.hidden = !show;